solr/core/src/java/org/apache/solr/cloud/autoscaling/OverseerTriggerThread.java

```diff
@@ -366,7 +366,6 @@ private AutoScalingConfig withDefaultTrigger(Map<String, Object> triggerProps, A
       }
     }
     // need to add
-    triggerProps.computeIfPresent("waitFor", (k, v) -> (long) (cloudConfig.getAutoReplicaFailoverWaitAfterExpiration() / 1000));
```

Comment:
We need to add a default value here instead of of outright removing this line. Use the default value which is in `CloudConfig.DEFAULT_AUTO_REPLICA_FAILOVER_WAIT_AFTER_EXPIRATION` and set to 2 minutes.